src/components/Input/Input.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styled from "styled-components/macro";
+
+const StyledInput = styled.input`
+  border-radius: 8px;
+  padding: 20px;
+  font-size: 16px;
+  outline: 0;
+  border: 2px solid #eaeaea;
+`;
+
+const Input = () => {
+  return <StyledInput />;
+};
+
+export default Input;
```

Comment:
The problem is, that you did not declare any `Props` on `<Input />`. But you want it to have the props a normal `input` tag would have. 
The easiest way to solve this in this situation is to do the following:
```suggestion
const StyledInput = styled.input`
  border-radius: 8px;
  padding: 20px;
  font-size: 16px;
  outline: 0;
  border: 2px solid #eaeaea;
`;

export default StyledInput;
```